packages/nextjs/app/configure/_components/DownloadContracts.tsx

```diff
@@ -130,7 +130,7 @@ export default function DownloadContracts() {
               </div>
               <input
                 value={symbol}
-                onChange={handleInputChange}
+                onChange={(e) => setSymbol(e.target.value)}
```

Comment:
what symbol does this refer to?

Reply:
contract name

Reply:
might want to change it haha, it will confuse other ppl if they make changes or read the codebase